File: code_practice/hello_jack.py
# a code that implements the same function as hello_jack.c
name = input("What's your name?")
print("hello,", name) # 默认在字符串间加上空格
print("hello,",name, sep="?")
print(f"hello, {name}") #和上面输出形式等效
print("hello, "+ name) #和上面输出形式等效

File: code_practice/hello_jack.c
// hello_jack.c
// to run the c code, you need to prebuild make file 
// that link lncs50.library to C compilers.
// to run hello_jack.c, first insert in the terminal "make hello_jack"
// after successfully compiling, then insert "./hello_jack", no space in . and /
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    string name = get_string("What's your name?\n");
    printf("Hello, %s\n", name);
    
}